### ISSUE TITLE
Can't open account's settings

### ISSUE THREAD

**Issue by dmi7ry** [RocketChat/Rocket.Chat#26637]
### Description:

"My Account" shows black screen.

### Steps to reproduce:

1. Click your avatar
2. Click "My Account"

### Expected behavior:

Open account preferences

### Actual behavior:

There is just empty black screen

### Server Setup Information:

- Version of Rocket.Chat Server: 5.0.3 (also tried 4.8)
- Operating System: Ubuntu 20.04
- Deployment Method: CapRover (Docker)
- Number of Running Instances: 1
- DB Replicaset Oplog: 
- NodeJS Version: v14.19.3
- MongoDB Version: bitnami/mongodb:4.4 (4.4.15)

### Client Setup Information

- Desktop App or Browser Version: Edge 104.0.1293.54, Chrome, etc. App 3.8.7
- Operating System: Windows 10

### Relevant logs:

Browser's console shows:

    ...b7b63.js?meteor_js_resource=true:1124 TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))

**Comment by debdutdeb** [comment]
Is this happening only for admin account or all? Can you check that please?

**Comment by dmi7ry** [comment]
Yep, looks like only for admin. For another account it works fine.

**Comment by debdutdeb** [comment]
Open the mongo shell and run this 👇🏼 
```js
db.getSiblingDB("rocketchat").updateOne(
    {roles: "admin", username: "<your username>"},
    {$push: {emails: {address: "<your email address", verified: true}}},
    {upsert: false}
)
```

**Comment by dmi7ry** [comment]
Thank you for the tip! I was able to edit my profile in administration/users (added email + set it as verified) and it helped.

**Comment by debdutdeb** [comment]
Ah yes there is the admin settings. Lol forgot about that 😂 

I opened a fix for new installed on the caprover repo. 

Thanks for creating this issue! 😇
